Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2511,7 +2518,7 @@ extension BrowserViewController: WKUIDelegate {
             // disabled in settings, for example), we'll fail to load the session restore URL.
             // We rely on loading that page to get the restore callback to reset the restoring
             // flag, so if we fail to load that page, reset it here.
-            if AboutUtils.getAboutComponent(url) == "sessionrestore" {
+            if url.getAboutComponent() == "sessionrestore" {
```

Comment:
Just thinking out loud: it'd be nice if `getAboutComponent` returned an enum rather than a string. Can fix this in a separate bug though.